nav2_bringup/launch/gazebo_ros_simplebot.world

```diff
@@ -0,0 +1,410 @@
+<?xml version="1.0"?>
+<!--
```

Comment:
Does it make more sense to put this documentation in a README file in the directory? I wouldn't normally think to look inside a .world file for instructions.

Reply:
I agree with Carl, can you add a short README instead?

Reply:
Agree, the README file is still work in progress.